### PR TITLE
[WIP]add publish tgzArchive and Meta example

### PR DESCRIPTION
**Pull request by dingdaoyi**
This is only an example and does not comply with the integration specifications. You can refer to this example.

### CODE REVIEW

File: src/main/java/com/artipie/npm/TgzArchive.java [hunk 1]
```diff
@@ -66,6 +81,40 @@ public Completable saveToFile(final Path path) {
...
+            throw new IllegalStateException(String.format("'%s' file wasn't found", name));
+        } catch (final IOException exc) {
+            throw new UncheckedIOException(exc);
+        }
+    }
+
+    public Meta meta() {
+        return new Meta(
+                new NpmPublishJsonToMetaSkelethon(
+                        Json.createReader(new StringReader(this.file("package.json")))
```
Comment (dingdaoyi) [inline src/main/java/com/artipie/npm/TgzArchive.java:116]: The method of converting package.json to cli meta needs to be implemented.

**Comment by genryxy** [comment]
@dingdaoyi thanks for your PR. We started to work on implementing this functionality in #36

**Comment by 0crat** [comment]
Job `gh:artipie/npm-adapter#173` is not assigned, can't get performer

**Comment by 0crat** [comment]
Pull request [#173](https://github.com/artipie/npm-adapter/issues/173) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)

**Pull request closed without merging**